go/nbs/store.go

```diff
@@ -381,19 +380,12 @@ func (nbs *NomsBlockStore) Commit(current, last hash.Hash) bool {
 		return true
 	}
 
-	b := &backoff.Backoff{
```

Comment:
Should we do something here where we somehow pipe up a backoff error from Dynamo if we get one, and respond appropriately?